### PR TITLE
Ft/rex trading

### PR DESCRIPTION
**Pull request by mindsgn**
rext trading update

### CODE REVIEW

**Comment by mikeghen** [comment]
@mindsgn, theres a contract for this now: 0x0983c50085C81A7F41598Eb46a29001989102119

So this can be finished up

**Comment by samirsalem** [comment]
Build fails due to some syntax issues here FYI

**Comment by mindsgn** [comment]
@samirsalem not ready for review. I will fix the issues as from tomorrow I am unable to get to it at the moment.

**Review by mikeghen — CHANGES_REQUESTED** [review]
@mindsgn did you test this? Its not working for me, not showing a balance, and USDCx is missing from the list
Screen Shot 2022-02-16 at 1 55 36 PM

**Comment by mindsgn** [comment]
@mikeghen I will fix this asap.

**Comment by shreyaspapi** [comment]
@mindsgn might also need to add this uni swap sdk mentioned in this comment - https://github.com/Ricochet-Exchange/ricochet-frontend/issues/96#issuecomment-1015733950

**Comment by josealonso** [comment]
I just added a *do not merge* label so the CI tool can destroy the environment and release resources for other PRs.

**Comment by josealonso** [comment]
@mindsgn said he's working on this.

File: src/containers/main/TradeContainer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  const updateAmountOut = (e:any) => {
+    const validNumber = new RegExp(/^\d*\.?\d*$/);
+    if (validNumber.test(e.target.value)) {
+      setAmountOut(e.target.value);
+    }
+  };
+
+  const Swap = () => { 
+  };
```
Comment (shreyaspapi) [inline src/containers/main/TradeContainer/index.tsx:103]: @mindsgn Let me know if you need help with this function, I believe there are some examples in the uniswap sdk docs - https://docs.uniswap.org/sdk/guides/auto-router . It should be straight forward but it took me a while to understand this the first time.

Comment (mindsgn) [reply src/containers/main/TradeContainer/index.tsx:103]: @shreyaspapi yes please help me, I'm on the last step which is the swap function.

**Comment by mikeghen** [comment]
Im closing this, its been open for a month and not merged

**Pull request closed without merging**